### PR TITLE
Add popup-Window

### PR DESCRIPTION
**Pull request by Abdo9826**
Add HTML codes 
Add CSS styles 
Add Js codes

### CODE REVIEW

File: a.js [hunk 1]
```diff
@@ -27,4 +35,32 @@ limenu3.addEventListener('click', () => {
...
+  mobilewindow.style.display = 'block';
+});
+
+mobilebutton5.addEventListener('click', () => {
+  mobilewindow.style.display = 'block';
+});
+
+mobilebutton6.addEventListener('click', () => {
+  mobilewindow.style.display = 'block';
+});
```
Comment (youmari) [inline a.js]: - [ ] I see that you hardcoded the projects cards they should generate dynamically with `Javascript` in order to be easy for you to add more cards, so please try to implement that and  create the cards in javascript dynamically as  in the requirements 
image

File: a.css [hunk 2]
```diff
@@ -1300,4 +1379,62 @@ textarea {
...
+  }
+
+  .p2window {
+    height: 336px;
+    width: 424px;
+    left: 0;
+    top: 0;
+    font-family: 'Inter', serif;
+    font-size: 16px;
+    font-weight: 400;
```
Comment (youmari) [inline a.css:1423]: - [ ] I see that you are using a transparent background for your pop up window it should be white as in the Figma design so please try to fix that 

- [ ] Figma design 
image

- [ ] Yours 
image

**Review by youmari — CHANGES_REQUESTED** [review]
Hi @Abdo9826 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are 

## Highlights
✔️ No linter error
✔️ Great title and short summary

## Required Changes ♻️

- [ ] [OPTIONAL] You have a space gap in the headline please try to remove it in order to match the Figma design 
- [ ] 
image

- [ ] [OPTIONAL] please try to name your files a better name than `a` as you did that will make your project a bit messy and not understandable by other developers  

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 3]
```diff
@@ -58,59 +58,59 @@ <h3 class="h3">
...
       <section id="body2" class="body2">
-
+        
         <div id="sec2">
         <h2 class="s2h1">My Recent Works</h2>
         <hr class="hr1" />
       </div>
-
+      
+       <!-- <div class="mainbody"> -->
```
Comment (smunozmo) [inline index.html:67]: - [ ] Pop-up window is looking great! However, as the previous reviewer said, the content of the pop-up window and the works section should be created dynamically, this means that all the hardcoded info from line 66 to 253 should be deleted and it should be loaded using DOM manipulation with JavaScript. The goal is to have an array with all the data from the projects and create a function that adds all the HTML elements to the DOM. 

Please remove all the hardcoded info from the HTML and use JS to load it dynamically.

**Review by smunozmo — CHANGES_REQUESTED** [review]
Hi @Abdo9826 , 👋  

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights 👏 

- Great work using the right GitHub Flow.
- Your README looks professional, well done!
- Commits are descriptive, good work.
- Linters are working properly, great job!


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

- [OPTIONAL] It would be better to use meaningful names for your files, this way your repository will look more professional for recruiters and other developers.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._




Cheers and Happy coding! 🤓 

✅  Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

⚠️  **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 4]
```diff
@@ -58,46 +58,46 @@ <h3 class="h3">
...
         <div id="hed2">
-          <img class="none" id="img1" src="image/img1.png" alt="Multi-Post Stories image">
-          <img class="none2" src="image/placeholder.png" alt="Multi-Post Stories image" />
-          
-          <div id="hed2p2">
-          <h3 class="s3h3">Multi-Post Stories</h3>
+          <!-- <img class="none" id="img1" src="image/img1.png" alt="Multi-Post Stories image">
```
Comment (smunozmo) [inline index.html:69]: [OPTIONAL] Now that your content is created dynamically, it would be great to clean up your HTML file by removing any unused code.

**Review by smunozmo — APPROVED** [review]
Hi @Abdo9826, 👋 

## Highlights 👏 

- Great work using the right GitHub Flow.
- Your README looks professional, well done!
- Commits are descriptive, good work.
- Linters are working properly, great job!

Your project is complete!  There is nothing else to say other than... it's time to merge it! 🚀 
Very nice work, congratulations! 🎉 

### APPROVED! 💯  💯  💯 




Cheers and Happy coding! 🤓 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

✅  Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**